manage-data/lifecycle/index-lifecycle-management/tutorial-general-content-with-data-streams.md

```diff
@@ -73,13 +119,84 @@ PUT _ilm/policy/indextods
   }
 }
 ```
+:::
+::::
 
-For more information about lifecycle phases and available actions, check [Create a lifecycle policy](configure-lifecycle-policy.md#ilm-create-policy).
+For more information about lifecycle phases and available actions, refer to [Create a lifecycle policy](configure-lifecycle-policy.md#ilm-create-policy).
 
 
 ## Create an index template to apply the ingest pipeline and lifecycle policy [manage-general-content-with-data-streams-template]
 
-Create an index template that uses the created ingest pipeline and lifecycle policy:
+To use the created lifecycle policy, you configure an index template that uses it.
+When creating the index template, specify the following details:
+* the name of the lifecycle policy, which in our example is `indextods`
+* the ingest pipeline that enriches the data by adding the `@timestamp` field, which in our example is `ingest_time_1`
+* that the template is data stream enabled by including the `data_stream` definition
+* the index pattern, which ensures that this template will be applied to matching indices and in our example is `movetods` 
+
+You can create the template in {{kib}} or with the [create or update index template](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-index-template) API.
+
+::::{tab-set}
+:group: kibana-api
+:::{tab-item} {{kib}}
+:sync: kibana
+To create an index template in Kibana, complete these steps:
+
+1. Go to **Stack Management > Index Management**. In the **Index Templates** tab, select **Create template**.
+
+    ![Create template page](/manage-data/images/elasticsearch-reference-tutorial-ilm-rollover-general-content-create-template.png "")
+
+1. On the **Logistics** page: 
+    1. Specify the name of the template. For example `index_to_dot`.
+    1. Specify a pattern to match the indices you want to manage with the lifecycle policy. For example, `movetodos`.
+    1. Turn on the **Create data streams** toggle.
+    1. Set the [index mode](elasticsearch://reference/elasticsearch/index-settings/time-series.md) to **Standard**.
+1. Optional: On the **Component templates** page, use the search and filter tools to select any [component templates](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-component-template) to include in the index template. The index template will inherit the settings, mappings, and aliases defined in the component templates and apply them to indices when they're created.
```

Comment:
```suggestion
1. Optional: On the **Component templates** page, use the search and filter tools to select any [component templates](/manage-data/data-store/templates.md#component-templates) to include in the index template. The index template will inherit the settings, mappings, and aliases defined in the component templates and apply them to indices when they're created.
```
Maybe we should link to [here](https://www.elastic.co/docs/manage-data/data-store/templates#component-templates) for the non-API users.